$OpenBSD$

OpenBSD does not have prctl.h
sys/wait.h for WIFEXITED and other macros.

Index: Lib/System.cpp
--- Lib/System.cpp.orig
+++ Lib/System.cpp
@@ -17,10 +17,14 @@
 
 #include <cstdlib>
 #  include <unistd.h>
-#  if !__APPLE__ && !__CYGWIN__
+#  if !__APPLE__ && !__CYGWIN__ && !__OpenBSD__
 #    include <sys/prctl.h>
 #  endif
 
+#if __OpenBSD__
+#  include <sys/wait.h>
+#endif
+
 #include <dirent.h>
 
 #include <cerrno>
@@ -316,7 +320,7 @@ void System::terminateImmediately(int resultStatus)
  */
 void System::registerForSIGHUPOnParentDeath()
 {
-#if __APPLE__ || __CYGWIN__
+#if __APPLE__ || __CYGWIN__ || __OpenBSD__
   // cerr<<"Death of parent process not being handled on Mac and Windows"<<endl;
   // NOT_IMPLEMENTED;
 #else
